test/Microsoft.IdentityModel.TestUtils/TestTheoryData.cs

```diff
@@ -1,7 +1,13 @@
-﻿using Microsoft.IdentityModel.Tokens;
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.IO;
+using System.Net;
+using Microsoft.IdentityModel.Protocols;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
```

Comment:
looks like only usings changes to this file, is that correct? If these were required how was it building before, if they're not, why are they added?

Reply:
Removed unused package and added copy right